scripts/release.js

```diff
@@ -1,192 +0,0 @@
-    `
-It's not possible to release a new version from the master branch.
-Please checkout any other branch:
-  - develop: release new stable version
-  - xxxxxxx: release new beta version
-  `.red
-  );
-  return process.exit(1);
-}
-
-const strategy = currentBranch !== 'develop' ? 'beta' : 'stable';
-
-// called if process is aborted before publish,
-// nothing is pushed nor published remove local changes
-function rollback(newVersion) {
-  if (strategy === 'stable') {
-    shell.exec('git reset --hard origin master');
-    shell.exec('git checkout develop');
-  } else {
-    shell.exec('git reset --hard HEAD~1');
-  }
-
-  // remove local created tag
-  shell.exec(`git tag -d ${newVersion}`);
-}
-
-const shouldReleaseVersion = readline.keyInYN(
-  `This script will release a new ${strategy} version, continue?`.yellow
-    .underline
-);
-
-// user started the script by mystake?
-if (!shouldReleaseVersion) return process.exit(0);
-
-if (strategy === 'stable') {
-  // merge develop on master first
-  // and start release from master branch
-  shell.echo('');
-  shell.echo('Updating working tree'.blue);
-  shell.exec('git checkout master');
-  shell.exec('git pull origin master');
-  shell.exec('git fetch origin --tags');
-  shell.exec('git fetch origin develop');
-  shell.exec('git merge origin/develop');
-}
-
-// Check if next CHANGELOG.md is correct
-const { version: currVersion } = require('../package.json');
-shell.echo(
-  `
-- Current version is "${currVersion}"
-- Changelog will be generated only if a fix/feat/performance/breaking is found in git log
-- You must choose a new ve.rs.ion (semver) ${
-    strategy === 'beta' ? 'with -beta.x suffix' : ''
-  }
-  `.blue
-);
-
-shell.exec('conventional-changelog -p angular -u');
-
-if (!readline.keyInYN('Is the changelog correct?'.yellow.underline)) {
-  return process.exit(1);
-}
-
-shell.echo('');
-const newVersion = readline.question(
-  strategy === 'stable'
-    ? 'Please type the new chosen version '.blue.underline
-    : 'Please type the new chosen version (with -beta.x suffix)'.blue.underline
-);
-
-if (strategy === 'beta' && !/-beta\.\d{1,}$/.test(newVersion)) {
-  shell.echo(
-    'The version you specified is incorrect, for a beta you must include `-beta.x` suffix in the version number'
-      .red
-  );
-  return process.exit(1);
-}
-
-shell.echo('');
-shell.echo(`Bumping version to "${newVersion}"`.blue);
-
-//  replace package.json with next version
-shell.exec(`VERSION=${newVersion} node scripts/bump-package-version.js`);
-
-// install dependencies
-shell.echo('');
-shell.echo('Install dependencies'.blue);
-shell.exec('yarn cache clean');
-shell.exec('yarn');
-
-// build library
-shell.echo('');
-shell.echo(`Building Angular InstantSearch v${newVersion}`.blue);
-shell.exec('npm run build');
-
-// Update changelog
-shell.exec('conventional-changelog -p angular -i CHANGELOG.md -s');
-
-// Copy README.md and CHANGELOG.md to dist folder
-shell.exec('cp README.md CHANGELOG.md dist');
```

Comment:
Is this something we still should do with Ship.js?
We don't do this for IS.js and it's probably okay without the line.
https://docs.npmjs.com/files/package.json#files

Reply:
You're right, because weirdly we `cd` to `dist` for publishing: https://github.com/algolia/angular-instantsearch/blob/9ea22d03ac8056ed6a349c1f1613e3a715679620/scripts/release.js#L182

How would we handle that with Ship.js?

Reply:
I guess this would work:

```js
{
  publishCommand: ({ defaultCommand }) => `cd dist && ${defaultCommand}`;
}
```

Reply:
hmmm `cd dist` looks weird.
What if we add `dist` to `files` in `package.json`, and don't `cd dist` ?
Theoretically it should work, right?

Reply:
I'm not too confident changing the whole pipeline though. This is what's exported on [unpkg](https://unpkg.com/browse/angular-instantsearch@2.2.2/).

Reply:
That's because we rely on [`ng-packagr`](https://github.com/algolia/angular-instantsearch/blob/9ea22d03ac8056ed6a349c1f1613e3a715679620/scripts/build.sh#L9).

Reply:
Ah didn't know that. Yes, let's go with `cd dist && .. `